pims/image_sequence.py

```diff
@@ -102,6 +121,17 @@ def _get_files(self, path_spec):
             self._count = len(path_spec)
             return
 
+        if zipfile.is_zipfile(path_spec):
+            self.is_zipfile = True
```

Comment:
For readability reasons can you put a `self._is_zipfile = False` and `self._zipfile = None` in the `__init__`?  I also suggest making these 'private' attributes as these details should be mostly hidden from the users.